treerec/tskit/convert.c

```diff
@@ -46,16 +46,17 @@ typedef struct {
     size_t precision;
```

Comment:
and I didn't review the changes inside the tskit code since I assume they've been reviewed by others already, etc.; if there are any SLiM-specific changes made here, or if you *removed* any previously-existing SLiM-specific changes that we had in the code before, please call my attention to those for review...

Reply:
nope, this is now just a straight copy of the tskit files.

Reply:
ah, that's awesome